### PR TITLE
Jpy learn objectives

### PR DESCRIPTION
**Pull request by joycepyang**
I tried to distill each of these down to one "core objective" based on our discussion during the facilitation meeting about wanting to identify what was critical (the agent of change / indispensable) to each session, with the hope that it addresses the core 'function' of each session, and that additional sub points could be made with the 'topography' of how we plan to achieve the objective with more specifics. I've gone through half of them; if it looks like a useful approach / can move the project forward, please let me know and I can address the second half.

### CODE REVIEW

File: facilitation_guide/learning_objectives.rmd [hunk 1]
```diff
@@ -1,42 +1,51 @@
...
+CORE OBJECTIVE: To develop familiarity in accessing localized team data.
+
 1.	Demonstrate how to access and update the team’s patient data on the SharePoint site.  
 2.	Identify a specific patient in the team data.  
 3.	Describe how the team data are categorized into visits by CPT Codes.  
 4.	Specify the team’s preferred data for Modeling to Learn.  
 5.	Describe the limits of retrospective data review for guiding change.  
 
 Session 3. Our System  
+CORE OBJECTIVE: To simulate clinic units (stocks: e.g., appointments available, and flows: e.g., booking rate) in the Care Coordination model in order to improve team decision making.
```
Comment (branscombj) [inline facilitation_guide/learning_objectives.rmd]: The core objective of Session 3 might be to learn the notation and nomenclature of system dynamics diagrams in preparation for using the models. ?

File: facilitation_guide/learning_objectives.rmd [hunk 2]
```diff
@@ -1,42 +1,51 @@
...
+
 1.	Describe the strengths of simulation modeling for improving team decision-making.  
 2.	Define "stock" and "flow" as used in system dynamics.
 3.  Name the two primary "stocks" that are included in Care Coordination model.   
 4.	Identify key parameters (data inputs) in the Care Coordination model.  
 
 Session 4. Care Coordination I  
+CORE OBJECTIVE: To compare and contrast "balancing" and reinforcing" feedback loops affecting patient flow through care coordination appointments.
```
Comment (branscombj) [inline facilitation_guide/learning_objectives.rmd]: I might say "To recognize..." but otherwise this seems good.

**Comment by TomRust** [comment]
Jane,

What's the best way to include the "primary LO" for each session? It looks like Joyce has created a new LO for each (or rephrased an existing LO).  I had thought just to "bold" the primary LO.

What works best for our audience?

**Comment by branscombj** [comment]
Tom,

Long answer because I think it's worth clarifying the purpose and intended uses of "primary LOs". Here's my interpretation; I'm open to other takes on it.

I think it's a good thing to do, to call out the learnings that we feel are sine qua nons for the course or for each session: We can emphasize them in the Session Guide so that Facilitators know they're central and not to be skirted. For learners, I think they should probably be less transparent (or in your face). If a session has, say, 4 LOs, there really shouldn't be any hierarchy among them.

It's fine if we start IDing the "primary LOs" now and use the current All-LOs document to track them; but how that info gets communicated will vary for different purposes. Each session's LOs will be copied into the EES form for that session's accreditation review; and once approved they'll become part of the information seen by potential participants about the continuing ed opportunity. I am imagining talking in the Facilitation Manual/Training about how there are a few, absolutely central learnings, and how they can recognize those in the Session Guides. 

Thus right now, I think all we're doing is coding them for our own purposes. So I'd say bolding them is fine.

Does this make sense to others??

Jane

**Comment by joycepyang** [comment]
Hi Jane, 

Yes that makes sense to me; I was approaching the LO editing exercise as trying to identify the core LO(s) due to our conversation during the facilitation meeting when we were discussing whether there were critical mechanisms within each session that we wanted to highlight for facilitators to make sure not to miss.

Joyce

**Comment by dlkibbe** [comment]
This makes sense and I agree with Jane. For facilitation guide scripts, we could include brief text that emphasizes the most important learning outcome or learning goal (Today, we really want to ensure you walk away with an understanding of X) or could simply provide a consistent order of the objectives -- the order in which the content is covered or ordered by importance, whatever the group decides.

**Pull request closed without merging**